### PR TITLE
Changed UI

### PR DESCRIPTION
**Pull request by AryanShivva**
changed the ui of the onetime intro page into a nice design and also changed the splashscreen of the app.

## Fixes Issue
#51 

## Changes proposed
Succesfully changed the splash screen UI of the app.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot 2023-05-31 at 11 43 17 PM

### CODE REVIEW

**Comment by Suryansh1720001** [comment]
Could you please provide me with the demo video of the application?

**Comment by Suryansh1720001** [comment]
@AryanShivva,

Please rename the images based on their usage:

1. Kindly rename this drawable image appropriately.

image

---

2. Please note that this image contains a Canva watermark, which is not acceptable. Ensure that the image is free from any watermarks before using it.

image

Furthermore, once you have successfully resolved these issues, please send the pull request (PR) in a single commit by squashing them.

**Comment by Suryansh1720001** [comment]
@AryanShivva 
Any Update?

**Review by itssuryansh1 — CHANGES_REQUESTED** [review]
remove watermark

File: app/src/main/res/drawable/new_backround_aryan.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Vaibhav2002) [inline app/src/main/res/drawable/new_backround_aryan.xml:1]: can you rename this file to 'background`?

**Comment by Vaibhav2002** [comment]
> @AryanShivva Could you please provide me with the demo video of the application?

@AryanShivva please upload this.

**Comment by AryanShivva** [comment]
https://github.com/Taaveez/Taaveez-android/assets/124916476/33ff6aa0-2d74-4bbf-b4e4-750936565315

**Comment by Vaibhav2002** [comment]
> 
> https://github.com/Taaveez/Taaveez-android/assets/124916476/33ff6aa0-2d74-4bbf-b4e4-750936565315
> 
> 

Why isn't the splash screen shown the first time?

**Comment by AryanShivva** [comment]
According to the code already written the splash screen will not appear for the first time when we are opening the app.One time Activity screen will directly appear for the first time.

**Comment by Suryansh1720001** [comment]
@AryanShivva 
Please rename the file "new_background.xml" and remove any mention of your name. Additionally, to avoid copyright issues, please ensure that any images used are free of restrictions or consider using a paid image.


---
Could you please provide a different image or use a paid version of the image? The current image you shared still has visible watermark signs. Thank you.

**Comment by Vaibhav2002** [comment]
> According to the code already written the splash screen will not appear for the first time when we are opening the app.One time Activity screen will directly appear for the first time.

Hi @AryanShivva this is kinda wrong UI, because Splash Screen should be shown each and every time. 
Please update the code to show Splash Screen every time.

**Comment by Suryansh1720001** [comment]
no response

**Pull request closed without merging**